clang-tools-extra/test/clang-tidy/checkers/bugprone/exception-rethrow.cpp

```diff
@@ -0,0 +1,103 @@
+// RUN: %check_clang_tidy %s bugprone-exception-rethrow %t -- -- -fexceptions
+
+struct exception {};
+
+namespace std {
+  template <class T>
+  T&& move(T &x) {
```

Comment:
```suggestion
  T&& move(T &&x) {
```
The official signature is the one below.
```c++
template< class T >
typename std::remove_reference<T>::type&& move( T&& t ) noexcept;
```